Wikipedia/Code/YearInReviewCoordinator.swift

```diff
@@ -370,43 +370,63 @@ final class YearInReviewCoordinator: NSObject, Coordinator {
         return String.localizedStringWithFormat(format, readCount, numArticlesString, numLanguagesString)
     }
     
-    func personalizedYouReadSlideTitleV3(readCount: Int, minutesRead: Int) -> String {
+    func personalizedYouReadSlideTitleV3(readCount: Int) -> String {
+        let minutesRead = readCount * 3
```

Comment:
@l-olson1214 We should keep the `minutesRead` logic as it was, that comes from personalized user data.